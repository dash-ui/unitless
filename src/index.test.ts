import unitless from './index'

describe('unitless', () => {
  it('should pass', () => {
    expect(unitless).toMatchSnapshot()
  })
})
