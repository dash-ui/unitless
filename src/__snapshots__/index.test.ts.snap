// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unitless should pass 1`] = `
Object {
  "WebkitLineClamp": 1,
  "animationIterationCount": 1,
  "borderImageOutset": 1,
  "borderImageSlice": 1,
  "borderImageWidth": 1,
  "boxFlex": 1,
  "boxFlexGroup": 1,
  "boxOrdinalGroup": 1,
  "columnCount": 1,
  "columns": 1,
  "fillOpacity": 1,
  "flex": 1,
  "flexGrow": 1,
  "flexNegative": 1,
  "flexOrder": 1,
  "flexPositive": 1,
  "flexShrink": 1,
  "floodOpacity": 1,
  "fontWeight": 1,
  "gridColumn": 1,
  "gridColumnEnd": 1,
  "gridColumnSpan": 1,
  "gridColumnStart": 1,
  "gridRow": 1,
  "gridRowEnd": 1,
  "gridRowSpan": 1,
  "gridRowStart": 1,
  "lineHeight": 1,
  "msGridColumn": 1,
  "msGridColumnSpan": 1,
  "msGridRow": 1,
  "msGridRowSpan": 1,
  "opacity": 1,
  "order": 1,
  "orphans": 1,
  "stopOpacity": 1,
  "strokeDasharray": 1,
  "strokeDashoffset": 1,
  "strokeMiterlimit": 1,
  "strokeOpacity": 1,
  "strokeWidth": 1,
  "tabSize": 1,
  "widows": 1,
  "zIndex": 1,
  "zoom": 1,
}
`;
